requirements-ci0.txt

```diff
@@ -0,0 +1,4 @@
+pennylane>=0.32
+qiskit
```

Comment:
We don't have to pin it here?

Reply:
This is the CI file for the tests that we *want* to run with the latest qiskit, so not here